src/api/article/collection/model.tests.js

```diff
@@ -32,15 +34,23 @@ describe('ArticleCollection model', () => {
     expect(errorHandler).to.have.been.called();
   });
 
-  it('should fail to save collection | dup slug', async () => {
+  // flaky! 🙅
```

Comment:
"duplicate" tests are always flaky😩
my guess is that mongodb needs more time to get it all indexed

Reply:
no, I fixed it with `.ensureIndexes()` for all cases
but then it breaks for collection after I add `unique: true` for `fiberyId` & `fiberyPublicId`. WTF 😠